src/server/pps/server/api_server.go

```diff
@@ -3008,7 +3008,7 @@ func (a *apiServer) RunLoadTest(ctx context.Context, req *pfs.RunLoadTestRequest
 			fmt.Sprintf("cp -r /pfs/%s/* /pfs/out/", repo),
 		},
 		&pps.ParallelismSpec{
-			Constant: 1,
+			Constant: 5,
```

Comment:
Did you mean to leave this changed?

Reply:
Yeah, I wanted to bump the load test parallelism a bit.